recipes/realphy/meta.yaml

```diff
@@ -24,8 +24,7 @@ test:
     - realphy -help
     - realphy -version
 
-about:
-  home: 'https://realphy.unibas.ch/fcgi/realphy'
+about: 
+  home: 'https://realphy.unibas.ch/realphy'
   license: No commercial use
   summary: 'The Reference sequence Alignment based Phylogeny'
-  license_file: LICENSE
```

Comment:
Please add this back in

Reply:
ah right, my bad.
It's still confusing because the LICENSE file in the zip is the GPL, which is not in the github repo. I've just made a PR on https://github.com/fredeBio/REALPHY which adds the GPL also there: https://github.com/fredeBio/REALPHY/pull/4

Reply:
Oh, that's very confusing then. Maybe we should wait a bit to see if they actually intended to ship the GPL with it.